models/comments.model.js

```diff
@@ -10,4 +11,22 @@ exports.selectAllComments = (article_id) => {
    })
     }
 
+exports.addComments = (article_id, author, body) => {
+  
+   const query = 
+   `INSERT INTO comments (article_id, author, body) 
+   VALUES ($1, $2, $3)
+   RETURNING *;`;
+      
+   return db.query(query, [article_id, author, body])
+   .then((result) => {
+      return result.rows[0]
+   })
+   .catch((err)=> {
+      throw err
+  })
+ 
```

Comment:
I have two questions about this catch block
1. Do we need a catch block here ? 👀 
2. If we `throw` an error, nothing gets sent back to the client. What do we need to do instead?